examples/quickstart-pennylane/pyproject.toml

```
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "quickstart_pennylane"
+version = "1.0.0"
+description = "Quantum Federated Learning with PennyLane and Flower"
+license = "Apache-2.0"
+authors = [
+    { name = "SECQUOIA Research Group", website = "https://secquoia.github.io/" },
+    { name = "David Bernal Neira", email = "dbernaln@purdue.edu" },
+    { name = "Daniel Anoruo", email = "danoruo1@students.towson.edu" },
+    { name = "Alan Yi", email = "yi161@purdue.edu" }
+]
+dependencies = [
+    "flwr[simulation]>=1.21.0",
```

```suggestion
    "flwr[simulation]>=1.22.0",
```

Let's bump to the latest release 👼